src/IssueCreator/IssueManager.cs

```diff
@@ -281,9 +304,11 @@ internal async Task<(bool, string)> TryCreateNewIssueAsync(IssueToCreate issueTo
             return (true, string.Empty);
         }
 
-        public async Task<IssueObject> GetIssueAsync(long repoId, int issueNumber)
+        public async Task<IssueObject> GetIssueAsync(long repoId, int issueNumber, IssueLoadScenario loadScenario)
```

Comment:
I thought about this some more and was wondering if it makes sense to have 3 different methods for the 3 different enum values as 'convenience' methods.

Something like:
```csharp
public async Task<IssueObject> GetAllIssueAsync(long repoId, int issueNumber) => GetIssueAsync(repoId, issueNumber, IssueLoadScenario.LoadAllIssues);

public async Task<IssueObject> GetTemplateIssueAsync(long repoId, int issueNumber) => GetIssueAsync(repoId, issueNumber, IssueLoadScenario.LoadIssueAsTemplate);

public async Task<IssueObject> GetBrowseableIssueAsync(long repoId, int issueNumber) => GetIssueAsync(repoId, issueNumber, IssueLoadScenario.BrowseEpic);
```

That way it's clear which one you are using at each call site and you won't accidentally call the wrong method later on?

Reply:
I was being lazy, but yes, I agree 😃